js/app/dashboard/app.js

```diff
@@ -40,28 +40,35 @@ const drake = dragula({
 /**
  * Code to handle drag and drop from unassigned tasks to assigned
  */
-function configureOnDrop(allTasks, assignTasks) {
+function configureDrake(unassignedTasksContainer, allTasks, assignTasks) {
 
   drake
+    .on('dragend', function (el) {
+      Array.from(unassignedTasksContainer.querySelectorAll('.task__draggable--dragging'))
+        .forEach(el => el.classList.remove('task__draggable--dragging'))
+    })
     .on('drop', function(element, target, source) {
 
       const username = $(target).data('username')
-      let tasks = []
-
-      if ($(element).data('task-group') === true) {
-        tasks = $(element)
-          .children()
-          .map((index, el) => $(el).data('task-id'))
-          .map((index, taskID) => _.find(allTasks, task => task['@id'] === taskID))
-          .toArray()
-      } else {
-        const task = _.find(allTasks, task => task['@id'] === $(element).data('task-id'))
-        tasks.push(task)
+      const isTask = element.hasAttribute('data-task-id')
+      const elements = isTask ? [ element ] : Array.from(element.querySelectorAll('[data-task-id]'))
+
+      let tasks = elements.map(el => _.find(allTasks, task => task['@id'] === el.getAttribute('data-task-id')))
+
+      // Make sure linked tasks are assigned too
```

Comment:
Nice to have comment here - I think you should add more comments to your code ;)

<strike>I am not sure this code belongs here - shouldn't it be in Redux?</strike>

Reply:
This can't be in Redux: the `assignedTasksByUser` reducers only "knows" about the assigned tasks, not all of them. 
Ok for the comments ✅